app/models/message.rb

```diff
@@ -86,7 +86,7 @@ class Message < ApplicationRecord
 
   after_create_commit :execute_after_create_commit_callbacks
 
-  after_update :dispatch_update_event
+  after_update_commit :dispatch_update_event
```

Comment:
@sojan-official Does this change has any unintended side effect?

The issue here was that the feedback text was not updated as the job may be fired even before the update.

Reply:
👍 Makes more sense to trigger the event after committing the changes to the database. Looks good to me.